### PR TITLE
0.0.2 bugfixes

### PR DESCRIPTION
**Pull request by seanjnkns**
1. Added protection to ensure masternode.service migration issues a restart post-upgrade.
2. Drop geth.out parsing from utils.
3. Pylint fixes and resolve several general Exception catch-alls.

Ian, even though this looks like a big bunch of changes, it's not really.  If you decide to make this into a 0.0.3 release, let me know so I can, or you can, update the versions.json and the VERSION= strings in akroma-mn-*.py.

### CODE REVIEW

**Comment by seanjnkns** [comment]
Ian, I'm going to get the compatibility matrix built into the software to prevent users from running it in unsupported OS/versions.  Should have that done later today, then I'll push the changes to make a 0.0.3 release.

**Comment by seanjnkns** [comment]
need to fix a few more items, but really close for another version cut.

**Comment by seanjnkns** [comment]
Files available at https://drive.google.com/open?id=1TLNnFSqfKmXnmH7NyDokquz8su7IkB6F

d9bff26f97aa101fc7bb3aad0faa46f8  akroma-mn-setup.x86_64
09b435162850a8385b88a0dc4a31e285  akroma-mn-utils.x86_64

In case you're still trying to get Travis to work.

**Contributor**
Address: 0xa2E85B0b8f7ed1569b5bBaf5aF74D07B65E0d538
Discord: seanjnkns#5616

**Comment by seanjnkns** [comment]
Files available at https://drive.google.com/open?id=1TLNnFSqfKmXnmH7NyDokquz8su7IkB6F

c1ddc8fa8542b9467c32c3e841069dc7  akroma-mn-setup.i686
79d7b8e35f5692f721e272e9ae29f0e8  akroma-mn-setup.x86_64
97c4cab1bd77d7d54bb7784723328a3d  akroma-mn-utils.i686
9632e299a25c77240554d05441bf8238  akroma-mn-utils.x86_64

In case you're still trying to get Travis to work.

**Contributor**
Address: 0xa2E85B0b8f7ed1569b5bBaf5aF74D07B65E0d538
Discord: seanjnkns#5616

File: source/akroma-mn-setup.py [hunk 1]
```diff
@@ -39,20 +46,23 @@ def main():
 
     # Display script version
     if args.version:
-        print("Version: %s" % VERSION)
+        print "Version: %s" % VERSION
         sys.exit(0)
 
-    os_family, arch = utils.os_detect() # Get the OS family (ie, Debian or RedHat) and machine architecture
+    os_name, os_family, os_ver, os_arch = utils.os_detect() # Get the OS, OS family (ie, Debian or RedHat),  OS version, and machine architecture
+    if os_name not in COMPAT_MATRIX or os_ver not in COMPAT_MATRIX[os_name]:
```
Comment (iandesj) [inline source/akroma-mn-setup.py]: This is great.

**Pull request merged**